### PR TITLE
[IOTDB-2264] load tsfile with aligned timeseries failed

### PR DESCRIPTION
**Pull request by yschengzi**
load a tsfile with align timeseries and find message like:
./sbin/start-cli.sh -e "load './data_for_load' autoregister=false"
Msg: 411: Cannot load file /data/iotdb/db_master_0104/./data_for_load/1641288670353-1-0-0.tsfile because root.lz.dev.person. is not a legal path

log：
2022-01-05 11:10:04,994 [pool-12-IoTDB-RPC-Client-2] ERROR o.a.i.d.q.e.PlanExecutor:1210 - fail to load file 1641288670353-1-0-0.tsfile
org.apache.iotdb.db.exception.metadata.IllegalPathException: root.lz.dev.person. is not a legal path
at org.apache.iotdb.db.metadata.utils.MetaUtils.splitPathToDetachedPath(MetaUtils.java:60)
at org.apache.iotdb.db.metadata.path.PartialPath.<init>(PartialPath.java:80)
at org.apache.iotdb.db.qp.executor.PlanExecutor.loadNewTsFileVerifyMetadata(PlanExecutor.java:1228)
at org.apache.iotdb.db.qp.executor.PlanExecutor.loadFile(PlanExecutor.java:1169)
at org.apache.iotdb.db.qp.executor.PlanExecutor.loadDir(PlanExecutor.java:1138)
at org.apache.iotdb.db.qp.executor.PlanExecutor.operateLoadFiles(PlanExecutor.java:1108)
at org.apache.iotdb.db.qp.executor.PlanExecutor.processNonQuery(PlanExecutor.java:322)
at org.apache.iotdb.db.service.basic.BasicServiceProvider.executeNonQuery(BasicServiceProvider.java:257)
at org.apache.iotdb.db.service.thrift.impl.TSServiceImpl.executeNonQueryPlan(TSServiceImpl.java:1900)
at org.apache.iotdb.db.service.thrift.impl.TSServiceImpl.executeNonQueryStatement(TSServiceImpl.java:1049)
at org.apache.iotdb.db.service.thrift.impl.TSServiceImpl.executeUpdateStatement(TSServiceImpl.java:1042)
at org.apache.iotdb.db.service.thrift.impl.TSServiceImpl.executeStatement(TSServiceImpl.java:552)
at org.apache.iotdb.service.rpc.thrift.TSIService$Processor$executeStatement.getResult(TSIService.java:2733)
at org.apache.iotdb.service.rpc.thrift.TSIService$Processor$executeStatement.getResult(TSIService.java:2713)
at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:38)
at org.apache.thrift.TBaseProcessor.process(TBaseProcessor.java:38)
at org.apache.thrift.server.TThreadPoolServer$WorkerProcess.run(TThreadPoolServer.java:248)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)

fix:

- During metadata verify, if there is no measurement id in a chunk, ignore this chunckMetadata, and compare other value chunkMetadata
- During autoregister schema, if there is no measurement id in a chunk, collect all type, encoding type and other messages, use IoTDB.metaManager.createAlignedTimeSeries to create a new aligned timeseries

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java [hunk 1]
```diff
@@ -1224,6 +1226,9 @@ private void loadNewTsFileVerifyMetadata(TsFileSequenceReader tsFileSequenceRead
         continue;
       }
       for (TimeseriesMetadata metadata : entry.getValue()) {
+        if (metadata.getMeasurementId().equals("")) {
+          continue;
+        }
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java:1231]:
```suggestion
        if ("".equals(metadata.getMeasurementId())) {
          continue;
        }
```

File: server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java [hunk 2]
```diff
@@ -1271,6 +1279,10 @@ private void createSchemaAutomatically(
         existSeriesSet.add(path.getMeasurementAlias());
       }
       for (ChunkMetadata chunkMetadata : chunkGroupMetadata.getChunkMetadataList()) {
+        if (chunkMetadata.getMeasurementUid().equals("")) {
+          isAligned = true;
+          continue;
+        }
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java:1285]:
```suggestion
        if ("".equals(chunkMetadata.getMeasurementUid())) {
          isAligned = true;
          continue;
        }
```

**Comment by HTHou** [comment]
Plz add some IT.

File: server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java (hunk 1 above)
Comment (yschengzi) [reply server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java]: fixed

File: server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java (hunk 2 above)
Comment (yschengzi) [reply server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java]: fixed

**Comment by yschengzi** [comment]
> Plz add some IT.

fixed

File: integration/src/test/java/org/apache/iotdb/db/integration/aligned/IoTDBLoadExternalAlignedTsFileIT.java [hunk 3]
```diff
@@ -0,0 +1,829 @@
...
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class IoTDBLoadExternalAlignedTsFileIT {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(IoTDBLoadExternalTsfileIT.class);
```
Comment (HTHou) [inline integration/src/test/java/org/apache/iotdb/db/integration/aligned/IoTDBLoadExternalAlignedTsFileIT.java:59]:
```suggestion
  private static final Logger LOGGER = LoggerFactory.getLogger(IoTDBLoadExternalAlignedTsFileIT.class);
```

Comment (yschengzi) [reply integration/src/test/java/org/apache/iotdb/db/integration/aligned/IoTDBLoadExternalAlignedTsFileIT.java]: fixed

**Review by HTHou — APPROVED** [review]
LGTM

**Pull request merged**